.env.production.sample

```diff
@@ -160,6 +160,12 @@ STREAMING_CLUSTER_NUM=1
 # Maximum number of pinned posts
 # MAX_PINNED_TOOTS=5
 
+# Maximum image and video upload sizes
+# Units are in bytes
+# 1024 bytes equals 1 megabyte
```

Comment:
1024 bytes equals 1 kilobyte though

Reply:
Oh damn that's my bad lol I know this, just wasnt thinking. Is fixed now.